spec.html

```diff
@@ -43383,6 +43414,44 @@ <h1>The Strict Mode of ECMAScript</h1>
   </ul>
 </emu-annex>
 
+<emu-annex id="sec-host-layering-points">
+  <h1>Host Layering Points</h1>
+  <p>See <emu-xref href="#sec-hosts-and-implementations"></emu-xref> for the definition o host.</p>
+  <emu-annex id="sec-host-hooks-summary">
+    <h1>Host Hooks</h1>
+    <p><b>HostEnqueuePromiseJob(...)</b>: See <emu-xref href="#sec-hostenqueuepromisejob"></emu-xref></p>
+    <p><b>HostEnsureCanCompileStrings(...)</b>: See <emu-xref href="#sec-hostensurecancompilestrings"></emu-xref></p>
+    <p><b>HostFinalizeImportMeta(...)</b>: See <emu-xref href="#sec-hostfinalizeimportmeta"></emu-xref></p>
+    <p><b>HostGetImportMetaProperties(...)</b>: See <emu-xref href="#sec-hostgetimportmetaproperties"></emu-xref></p>
+    <p><b>HostHasSourceTextAvailable(...)</b>: See <emu-xref href="#sec-hosthassourcetextavailable"></emu-xref></p>
```

Comment:
FYI I can't find any definition of HostHasSourceTextAvailable in HTML; instead hosts use CreateBuiltinFunction to create a function which automatically has no source text. I don't remember the history here...

Reply:
HostHasSourceTextAvailable is pretty new, and iirc was added so engines like xs have a way to shut off source text for all functions.